core/templates/components/ck-editor-helpers/ck-editor-copy-content-service.ts

```diff
@@ -153,6 +158,16 @@ export class CkEditorCopyContentService {
     this.copyModeActive = !this.copyModeActive;
   }
 
+  // Function to check if the parent element or the current element is
+  // ANGULAR_HTML_BIND or not.
+  isElementParentElementOutputView(target: HTMLElement): boolean {
```

Comment:
Name it `isElementOrParentElementOutputView` in that case.

Reply:
Changed the name! It should make sense now.